src/components/SensorMap/SensorMap.js

```diff
@@ -30,33 +30,12 @@ export const SensorMap = () => {
     <Container className='my-3 p-0'>
       <Row>
         <Col className='p-0'>
-          {data.MeanClimateMeasurements.map((s) => (
-            <div key={s.floor + '-' + s.loc_x + '-' + s.loc_y}>
-              <img
-                className='sensor'
-                src={sensorImg}
-                style={{
-                  position: 'absolute',
-                  width: CELL_WIDTH + '%',
-                  height: CELL_HEIGHT + '%',
-                  left: (s.loc_x - 1) * CELL_WIDTH + '%',
-                  top: (s.loc_y - 1) * CELL_HEIGHT + '%',
-                }}
-                alt='Sensor'
-              />
-              <span
-                className='sensor-measurement'
-                style={{
-                  position: 'absolute',
-                  left: s.loc_x * CELL_WIDTH + '%',
-                  top: (s.loc_y - 1) * CELL_HEIGHT + '%',
-                }}
-              >
-                <span className="map-temperature">
-                {s.temperature.toFixed(1) + '°C'}
-                </span>
-              </span>
-            </div>
+          {data.MeanClimateMeasurements.map((sensor) => (
+            <Sensor
+              {...sensor}
+              cellHeight={CELL_HEIGHT}
+              cellWidth={CELL_WIDTH}
```

Comment:
just move these with definition into the Sensor component

Reply:
I left them inside the map so that if a different grid is presented, it would make more sense to fill it in from the map rather than the sensor.

Reply:
Actually you're right, sorry